src/views/GithubCodePermalinkReferenceWidget.vue

```diff
@@ -181,6 +184,28 @@ export default {
 }
 </script>
 
+<!--style lang="scss">
```

Comment:
@julien-nc This works fine for me as soon as I comment out the code in beforeMount

Reply:
Oh wait no, was mislead by browser cache